.github/workflows/avm.ptn.authorization.policy-assignment.yml

```diff
@@ -20,6 +20,10 @@ on:
         description: "Remove deployed module"
         required: false
         default: true
+      customLocation:
```

Comment:
Hello there 👋 
Let's say this is a way to overcome excluded locations, but I don't think it would harm considering it can only work when manually triggered (I.e., when publishing is disabled)

Fine for me, but waited for all CARML modules to be merged, so I think latest published modules need to be added.

Reply:
Also, it would probably be useful to add an example in the field, to show which convention to use (e.g. West Europe or westeurope)

Reply:
Updated to 
![image](https://github.com/Azure/bicep-registry-modules/assets/5365358/f164f075-c384-46e4-8c42-844b9d10c784)

Reply:
I think this may be due to other PRs having merged again in the meantime. As @ChrisSidebotham pointed out, this PR runs high risk of running stale fast 😄 I'm trying to find out which are missing. Looking at all workflows it should be 132, not 130.

Reply:
Should be fixed now

Reply:
@eriqua, this is neither the time nor place to let people know that I did not pay attention in math class.

Reply:
According to a friend, the numbers should add up now